core/src/test/java/cz/o2/proxima/storage/StreamElementTest.java

```diff
@@ -91,6 +108,23 @@ public void testWildcardUpdateCreation() {
     assertFalse(element.isDeleteWildcard());
   }
 
+  @Test
+  public void testWildcardUpdateCreationWithSequentialId() {
+    long now = System.currentTimeMillis();
+    StreamElement element =
```

Comment:
Can we assert also value of SequentialId in new tests?

Reply:
We don't do that? That is unfortunate, thanks! :)